azure-quantum/azure/quantum/target/quantinuum.py

```diff
@@ -17,12 +17,15 @@ class Quantinuum(Target):
         "quantinuum.hqs-lt-s1",
         "quantinuum.hqs-lt-s1-apival",
         "quantinuum.hqs-lt-s1-sim",
+        "quantinuum.qpu.h1-1",
```

Comment:
It seems like only h1-1 targets are here, do you know why all the h1-2 versionso of the targets seem to be missing from this list?

Reply:
@ricardo-espinoza, do you have any context on why only h1-1 targets are in this target_name set?

Reply:
All the testing was done with what you see here -- only hqs-lt-s1 names were present before -- and cut-off for September QDK release is tomorrow morning.  Could we create a bug to add h1-2 names in a later release?

Reply:
@xinyi-joffre , @ricardo-espinoza ,
I added the h1-2 target names in azure-quantum/azure/quantum/target/quantinuum.py and in the unit tests.  Running the tests again.

Reply:
It seems that the related PR #371 hasn't been merged. If that had been merged earlier, then Warren's approach of just adding the new names where the old names were would have been enough.

Reply:
I would recommend we exclude the -2 targets until the related PR is merged. Other than that, LGTM.

Reply:
I removed the h1-2 target names.